syntax="proto3";

message RegisterC2S {
    string userName = 1;
    string password = 2;
}